types/redis-server/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for redis-server 1.2
+// Project: https://github.com/BrandonZacharie/node-redis-server
+// Definitions by: Sander Timmermans <https://github.com/Noldaru>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = RedisServer;
+export as namespace RedisServer;
```

Comment:
Documentation or code doesnt show this as UMD module https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
```suggestion
```